src/index.html

```diff
@@ -9,10 +9,56 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="./styles/style.css"
     />
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="body">
+    <div class="stars stars--0">
```

Comment:
In this case, you have duplicate element `<div class="stars__star"></div>`. For cases with duplication, it is better to use a list, not div.